rigex-front-end/src/components/navBar/NavBar.js

```diff
@@ -1,33 +1,27 @@
-import AppBar from '@material-ui/core/AppBar';
-import Toolbar from '@material-ui/core/Toolbar';
-import IconButton from '@material-ui/core/IconButton';
-import Button from '@material-ui/core/Button';
+import SearchIcon from '@material-ui/icons/Search';
+import PersonOutlineOutlinedIcon from '@material-ui/icons/PersonOutlineOutlined';
 
 import routePaths from '../../constants/routePaths';
 import logo from "../../assets/rigex-logo-small.svg";
-import search from "../../assets/search.svg";
-import account from "../../assets/account.svg";
 import "./navBar.scss"
 
 export default function NavBar() {
 
   return (
     <div className="navBar">
-      <AppBar position="sticky" color="inherit">
-        <Toolbar>
-          <Button href={routePaths.landing} color="inherit" >
-            <img src={logo} alt="Rigex" />
-          </Button>
-          <div className="iconBox">
-            <IconButton aria-label="search bar button" >
-              <img src={search} alt="search" className="icon"/>
-            </IconButton>
-            <IconButton aria-label="account of current user" >
-              <img src={account} alt="account" className="icon" />
-            </IconButton>
-          </div>
-        </Toolbar>
-      </AppBar>
+      <div className="logo">
+        <a href={routePaths.landing} >
```

Comment:
Is there any reason you are using `a` tags instead of react router's `Link` and link type tags?

Reply:
Link elements from react-router-dom weren't working, but we do have a history instance we're using elsewhere, so I was able to switch to using that in an onClick for a button.